symengine/complex_mpc.cpp

```diff
@@ -908,6 +908,24 @@ class EvaluateMPC : public Evaluate
                 MPFR_RNDN);
         return complex_mpc(std::move(t));
     }
+    virtual RCP<const Basic> floor(const Basic &x) const override
+    {
+        SYMENGINE_ASSERT(is_a<ComplexMPC>(x))
+        mpc_class t(down_cast<const ComplexMPC &>(x).as_mpc().get_prec());
+        mpc_div_fr(t.get_mpc_t(),
+                   down_cast<const ComplexMPC &>(x).as_mpc().get_mpc_t(),
+                   MPFR_RNDDD);
```

Comment:
@isuruf How can `MPFR_RNDDD` be made visible in `complex_mpc.cpp` ? I was unable to make out how the same was achieved for `MPFR_RNDN`.